configuration-as-code-examples/cloudbees-assurance-plugin/casc_enabled.yaml

```diff
@@ -0,0 +1,11 @@
+# This file shows how to enable the CloudBees Assurance Program using Configuration-as-Code.
+# With this configuration, the security warnings for the core and the plugins are enabled.
```

Comment:
By "for the core", do you mean CloudBees Core? It's not clear.

Reply:
No, it means the Jenkins core underlying our products. Do you think there's a better way of expressing it?